sopel_waifu/waifu.json5

```diff
@@ -8381,11 +8381,13 @@
         "Trigger",
         "Yixuan",
         "Ju Fufu",
+        "Ukinami Yuzuha",
+        "Alice Thymefield",
```

Comment:
```suggestion
        "Alice Thymefield",
        "Ukinami Yuzuha",
```

"Order by https://zzz3.hakush.in/character" it says above.

<img width="981" height="597" alt="Image" src="https://github.com/user-attachments/assets/a09bd4dd-0b6a-4de1-b9d0-ddb4a597297c" />